assets/js/Containers/FrevaGPT/index.js

```diff
@@ -152,9 +152,13 @@ class FrevaGPT extends React.Component {
   }
 
   async handleKeyDown(e) {
-    if (e.key === "Enter" && !isEmpty(e.target.value.trim())) {
+    if (e.keyCode === 13 && !e.shiftKey) {
```

Comment:
why did you changed `key` to `keyCode`? Keycode is a deprecated feature. I have now doubts that it will work for the next 20 years, but still

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

other than that I can't comment to much right now because the chatbot isn't working. As soon as I am able to see anything, I will give more feedback